pkg/lib/zip/zip.go

```diff
@@ -193,7 +193,8 @@ func addFileToZip(fileInput *FileInput, zipInput *Input, archive *zip.Writer, ad
 }
 
 func addDirToZip(dirInput *DirInput, zipInput *Input, archive *zip.Writer, addedPaths strset.Set) error {
-	paths, err := files.ListDirRecursive(dirInput.Source, true, dirInput.IgnoreFns...)
+	emptyExcludes := make([]string, 0)
```

Comment:
Will this also work if it's `var emptyExcludes []string` (i.e. if it's `nil`)? It seems to me like it will, just wanted to double check. Also using `nil` will allocate slightly less memory.

Reply:
Yep. Just tested.